kvm.go

```diff
@@ -259,7 +275,13 @@ func (d *Driver) validatePrivateNetwork() error {
 		"255.255.255.0",
 		"192.168.42.2",
 		"192.168.42.254")
-	network, err = d.conn.NetworkDefineXML(xml)
+
+	//TODO(aaron-prindle) see if this is needed?
+	conn, err = d.getConn()
```

Comment:
I don't think this is needed, since you're doing it on line https://github.com/dhiltgen/docker-machine-kvm/pull/18/files#diff-bda8ba8a54ef41ed9e8a4e3427ef653dR225